make/common/MakeBase.gmk

```diff
@@ -140,6 +140,17 @@ FindAllReposRel = \
 
 ################################################################################
 
+# If reproducible builds are enabled then set SOURCE_DATE_ISO_8601 string variable
+ifeq ($(ENABLE_REPRODUCIBLE_BUILD), true)
+  SOURCE_DATE_ISO_8601 := $(shell $(DATE) --utc --date="@$(SOURCE_DATE_EPOCH)" +"%Y-%m-%dT%H:%M:%SZ" 2> /dev/null)
+  ifeq ($(SOURCE_DATE_ISO_8601), )
+    # GNU date format did not work, try BSD date options
+    SOURCE_DATE_ISO_8601 := $(shell $(DATE) -u -j -f "%s" "$(SOURCE_DATE_EPOCH)" +"%Y-%m-%dT%H:%M:%SZ" 2> /dev/null)
```

Comment:
Could we maybe figure out the date command line to use in configure instead to avoid the trial and error at build time?

Reply:
I based this trial and error based on util.m4 code and copied the method, so yes I can store the result from then...

Reply:
Add IS_GNU_DATE variable